app/src/main/java/phone/vishnu/quotes/activity/MainActivity.java

```diff
@@ -102,6 +104,7 @@
 
     private CircularProgressIndicator progressIndicator;
     private ChipGroup chipGroup;
+    private SwipeListener swipeListener;
```

Comment:
I don't see a need for this. Just using `new SwipeListener(homeFAB);` would do, IMO